.github/workflows/architecture-book.yml

```diff
@@ -30,8 +33,8 @@ jobs:
 
       - name: Install mdBook
         run: |
-          cargo install --version ${MDBOOK_VERSION} mdbook
-          cargo install --version ${MDBOOK_LAST_CHANGED_VERSION} mdbook-last-changed
+          cargo install --version ${MDBOOK_VERSION} mdbook || echo "mdbook already exists"
+          cargo install --version ${MDBOOK_LAST_CHANGED_VERSION} mdbook-last-changed || echo "mdbook-last-changed already exists"
```

Comment:
`cargo install` errors because of cache, this prevents CI from failing in that case.